application/common/models/User.php

```diff
@@ -278,4 +309,20 @@ public function validateAuthKey($authKey)
     {
         return false;
     }
+
+    /**
+     * Get this user as an AuthUser object
+     * @return AuthUser
+     */
+    public function getAuthUser()
+    {
+        $authUser = new AuthUser();
+        $authUser->firstName = $this->first_name;
+        $authUser->lastName = $this->last_login;
```

Comment:
It looks like autocomplete caused a mistake here.  My guess is that you wanted last_name, not last_login.